okhttp-aws-signer/gradle.properties

```diff
@@ -0,0 +1,3 @@
+LIBRARY_VERSION=1
```

Comment:
This is a public repository and the latest published version is 1.0.1. Maybe we should make this 1.0.1 here too?

Reply:
yes I will change it but everyone who's using this library will have to change to `okhttp-aws-signer:okhttp-aws-signer` as well

Reply:
Could you adjust the README file too?